File: 2-semestre/uri-online-judge/java/1020.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int idadeDias, anos, meses, dias;
        
        Scanner entrada = new Scanner(System.in);
        
        idadeDias = entrada.nextInt();
        
        anos = idadeDias / 365;
        meses = (idadeDias % 365) / 30;
        dias = (idadeDias % 365) % 30;
        
        System.out.printf("%d ano(s)\n", anos);
        System.out.printf("%d mes(es)\n", meses);
        System.out.printf("%d dia(s)\n", dias);
    }
}

File: 2-semestre/uri-online-judge/c/1020.c
#include <stdio.h>

int main() {
    int idadeDias, anos, meses, dias;
    
    scanf("%d", &idadeDias);
    anos = idadeDias / 365;
    meses = (idadeDias % 365) / 30;
    dias = (idadeDias % 365) % 30;
    
    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);
    
    return 0;
}
